README.md

```diff
@@ -63,42 +64,46 @@ npm run serve
 ## Formatting
 
 ### Code formatting
+
 Code and response examples should be inside code blocks with proper language:
+
 ````md
 ```graphql
-query{
+query {
   id
   name
 }
-``` 
+```
 ````
 
 ````md
 ```json
 {
   "errorCode": 400
 }
-``` 
+```
 ````
 
 ### Lining pages
+
 Use full path to the file to avoid linking to wrong page.
 
 - :white_check_mark: Example of good link: `[Attributes](/docs/developer/attributes.mdx)`
 - :stop_sign: Avoid: `[Attributes](/attributes)`
 
 ### Using custom React components
+
 All documentation files use extension:
+
 - `.mdx` - Developer documentation
 - `.md` - Dashboard documentation
 
 If your page uses custom react components, you are required to use `.mdx` file extension. Import statement is also required:
 
 ```mdx
-<!-- /docs/developer/export/export-overview.mdx file -->
----
-title: Exporting Products
----
+## <!-- /docs/developer/export/export-overview.mdx file -->
+
+## title: Exporting Products
```

Comment:
Are we sure this is correct? It looks like it turned both the comment and the meta block into a header.